mods/tnt/API.txt

```diff
@@ -0,0 +1,24 @@
+tnt.register_tnt(definition) Register a new type of tnt.
+
+definition
+{
+	name = "", Add a prefix to the beginning to use a namespace other then tnt.
+	description = "", A description for your tnt.
+	radius = 3, The radius in which tnt can damage nodes.	
+	damage_radius = 6, The radius within which tnt can damage players and mobs. By default twice radius.
+	disable_drops = false, Disable drops. 
+	tiles={
+	side = "", side tiles. By default name_side.png
+	top = "", top tile. By default name_top.png
+	bottom = "", bottom tile. By default name_bottom.png
+	burning = "", top tile when burning. By default name_top_burning_animated.png
+	
+	},
+	
+}
+
+tnt.boom(position, radius, damage_radius, disable_drops) Create an explosion.
+position- centrer of explosion
+radius - distance form centrer that block can get damaged in.
+damage_radius - distance form centrer that players and mobs can get damaged in.
+disable_drops - Disable drops.
```

Comment:
This should be in Markdown format.  The Lua code blocks should use comment characters(`--`).  There's an extra line fefore the last `}`.